### PR TITLE
text detection 

### PR DESCRIPTION
**Pull request by GalyaZalesskaya**

### CODE REVIEW

**Comment by GalyaZalesskaya** [comment]
@Ilya-Krylov  Done with docs, config, scripts for dataset convertation and metrics calculation.

File: pytorch_toolkit/object_detection/README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Models that are able to detect faces on given images.
 
 Models that are able to detect 3 classes of objects: person, vehicle, non-vehicle (e.g. bike).
 
+### [Horizontal Text Detection](text_detection.md)
+
+Model that is able to detect more or less horizontal text with high speed on CPU.
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/README.md:17]: Please remove CPU

File: pytorch_toolkit/object_detection/text_detection.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+### 3. Convert Datasets
+
+Extract downloaded datasets in `data/text-dataset` folder.
+
+Convert it to format that is used internally and split to the train and test part.
+
+* Training annotation
+```bash
+python3 tools/create_dataset.py \
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/text_detection.md:37]: is it proper path to create_dataset?

File: pytorch_toolkit/object_detection/text_detection.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    --config text-dataset/dataset_train.json \
+    --output data/IC13TRAIN_IC15_IC17_IC19_MSRATD500_COCOTEXT.json
+```
+* Testing annotation
+```bash
+python3 tools/create_dataset.py \
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/text_detection.md:43]: is it proper path to create_dataset?

**Comment by Ilya-Krylov** [comment]
please rebase on target to https://github.com/opencv/openvino_training_extensions/tree/feature/train_eval_face_detection

**Comment by Ilya-Krylov** [comment]
Add tests like in https://github.com/opencv/openvino_training_extensions/tree/develop/pytorch_toolkit/object_detection/tests

**Comment by Ilya-Krylov** [comment]
@GalyaZalesskaya please rebase on develop

File: pytorch_toolkit/object_detection/tools/gen_markdown_table.py [hunk 4]
```diff
@@ -1,101 +0,0 @@
-# Copyright (C) 2020 Intel Corporation
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/tools/gen_markdown_table.py:1]: why?

File: pytorch_toolkit/object_detection/requirements.txt [hunk 5]
```diff
@@ -3,3 +3,4 @@
 networkx==2.3
 opencv-python==4.1.0.25
 tensorboardX==1.8
+Polygon3==3.0.8
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/requirements.txt:6]: this is already in mmdetection/requirenments/runtime.txt that is installed before see 2nd line of this file

**Comment by Ilya-Krylov** [comment]
@GalyaZalesskaya please update mmdetection commit, currently commit hash points to non-existing commit.

File: pytorch_toolkit/object_detection/readme.md [hunk 6]
```diff
@@ -6,6 +6,7 @@ This repo contains scripts and tutorials for object detection models training.
 
 * [Face Detection](face-detection/readme.md) - models that are able to detect faces on given images.
 * [Person Vehicle Bike Detection](person-vehicle-bike-detection/readme.md) - models that are able to detect 3 classes of objects: person, vehicle, non-vehicle (e.g. bike).
+* [Horizontal Text Detection](text_detection.md) - model that is able to detect more or less horizontal text with high speed.
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/readme.md:9]: text-detection/readme.md

**Comment by Ilya-Krylov** [comment]
run tests

File: pytorch_toolkit/object_detection/horizontal-text-detection/problem.yml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+description:  >-
+  Face detection is a computer technology being used in a variety of applications
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/horizontal-text-detection/problem.yml:2]: It is not face detection

**Pull request merged**